src/AWS/Shared/Storage/DynamoDBStorage.cs

```diff
@@ -58,6 +61,7 @@ internal class DynamoDBStorage
             string accessKey = "",
             string secretKey = "",
             string token = "",
+            string profileName = "",
```

Comment:
Could we just pass the options class instance rather than having to grab and pass all of the properties from it individually?

Reply:
We can and I'm happy to do it after this PR reaches conclusion. I wouldn't want to propose such refactoring while we're still adding options.